src/json/Lookup.hxx

```diff
@@ -43,56 +43,56 @@ namespace Json {
 static inline const auto *
 Lookup(const boost::json::object &o, std::string_view key) noexcept
 {
-	return o.if_contains(key);
+  return o.if_contains(key);
```

Comment:
Do not change coding style of imported code. That way, you can't keep it in sync with upstream.

Reply:
Ok, this means, I have every time to fight with my editor - because it marks every tab in the code... And this I don't like to change ;-(

Reply:
Get a better editor. Mine can easily use different coding styles in different projects and even individual source files.

Reply:
Ok, good for you ... But I don't like to change everything... That's how I have to live with it!

Reply:
Now I have removed this commit